SnoopCompileCore/src/SnoopCompileCore.jl

```diff
@@ -23,4 +23,8 @@ if VERSION >= v"1.6.0-DEV.1192"  # https://github.com/JuliaLang/julia/pull/37136
     include("snoopl.jl")
 end
 
+if VERSION >= v"1.6.0-DEV.1192"  # https://github.com/JuliaLang/julia/pull/37136
+    include("snoop_all.jl")
```

Comment:
Can't we just include this macro into `SnoopCompileCore/src/snoopl.jl`? It's the same condition for including it that we use for`@snoopl`.

Reply:
Yeah, good idea. I've just moved it into the same if-block. I guess it doesn't necessarily belong in `snoopl.jl` because it will snoop all three types of compilation, which is why I put it in its own file.

Can you have another look and let me know if this looks okay? :)